.github/workflows/on_workflow_dispatch__build_publish_release.yml

```diff
@@ -0,0 +1,49 @@
+name: on_workflow_dispatch__build_publish_release
```

Comment:
I decided to split this up in two files to allow easier management. We can merge it later if we need to.